tests/test_utils.py

```diff
@@ -49,3 +49,15 @@ def test_ignores_non_prefixed_values(self):
 
     def test_takes_prefixed_values(self):
         self.assertDictEqual({'test': 1, 'something_else': 'test'}, collect_parameters_from_request({'p_test': 1, 'p_something_else': 'test'}))
+
+
+class TestCleanFromHtmlTags(TestCase):
+    def test_removes_html_tags(self):
+        dirty_string = '<a href="some/site" target="_brank">value</a>'
+        self.assertEqual('value', clean_from_html_tags(dirty_string))
+
+    def test_ignores_number(self):
+        self.assertEqual(3, clean_from_html_tags(3))
+
+    def test_ignores_string_with_no_html_tag(self):
+        self.assertEqual('value', clean_from_html_tags('value'))
```

Comment:
Good catch: haven't tried yet but I bet it raises some malformed html error
...

On Thu, 26 May 2016, 22:05 Arik Fraimovich, <notifications@github.com>
wrote:

> In tests/test_utils.py
> <https://github.com/getredash/redash/pull/1070#discussion_r64811559>:
>
> > @@ -49,3 +49,15 @@ def test_ignores_non_prefixed_values(self):
> >
> >      def test_takes_prefixed_values(self):
> >          self.assertDictEqual({'test': 1, 'something_else': 'test'}, collect_parameters_from_request({'p_test': 1, 'p_something_else': 'test'}))
> > +
> > +
> > +class TestCleanFromHtmlTags(TestCase):
> > +    def test_removes_html_tags(self):
> > +        dirty_string = '<a href="some/site" target="_brank">value</a>'
> > +        self.assertEqual('value', clean_from_html_tags(dirty_string))
> > +
> > +    def test_ignores_number(self):
> > +        self.assertEqual(3, clean_from_html_tags(3))
> > +
> > +    def test_ignores_string_with_no_html_tag(self):
> > +        self.assertEqual('value', clean_from_html_tags('value'))
>
> Will it handle this case: < whatever properly? (and return < whatever
> back)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/getredash/redash/pull/1070/files/3202559de4e6729aee41d3481dc7d4c7f51fbb9b#r64811559>
>